features/admins/admin_invitations.feature

```diff
@@ -398,6 +398,18 @@ Feature: Admin Actions to Manage Invitations
     Then I should see "oldman@ds9.com"
       And I should see "Invitation was successfully sent."
 
+  Scenario: Sent to field is a link for redeemed invitations
+    Given the user "creator" exists and is activated
+      And the user "invitee" exists and is activated
+      And an invitation created by "creator" and used by "invitee"
+      And I am logged in as a super admin
+    When I go to creator's manage invitations page
+    When I follow the first invitation token url
+    Then I should see "Sent to"
+      And I should see a link "default@email.com" within "dl"
+    When I follow "default@email.com"
+    Then I should see "All or part of an email address"
```

Comment:
Could you also check that we find the email within the `td` of the search result table?